druid/src/lens_ext.rs

```diff
@@ -0,0 +1,40 @@
+use crate::lens::Then;
```

Comment:
might it make sense to put this trait in `lens.rs` itself?

Another nit is that, in general, we try to have a docstring for each module. I think in druid we also include the license in each module file? I may have skipped this in an earlier review.

Reply:
Flattened into lens.rs.